### DISCUSSION TITLE
Upcoming Security Onion 2.3.60 Release

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#4434] [Announcements]
https://blog.securityonion.net/2021/06/upcoming-security-onion-2360-release.html

**Comment by TOoSmOotH** [comment]
I would also like to mention that if you have made customizations to any parts of the elastic stack that you will need to apply this update then reapply your changes. There have been a lot of config changes between 7.11 and 7.13.

**Reply by Jackson-Pollock** [reply]
ECS not yet in this release?  And incase SO become ECS compliant then all old queries(which has old fields name) will break?

**Reply by TOoSmOotH** [reply]
@eklike100 All of the new filebeat modules will be using full ECS in this release. We are testing the use of field aliases to address the transition for 2.3.70. Although ECS is a step in the right direction, there are still some gaps around pivoting that will need to be addressed. There is a possibility we will need to add some additional fields to certain logs and likely some additional tags. Once 2.3.60 drops we will communicate the changes of 2.3.70 ahead of time so that everyone is prepared for the transition.
